wiki/Article_styling_criteria/Formatting/en.md

```diff
@@ -794,7 +794,7 @@ The following is an example of what a table should look like:
 
 ## Blockquotes
 
-The blockquote is limited to quoting text from someone. It must not be used to format text otherwise. The name of the quoted person that follows the quote must be prepended by an em dash (`—`) as follows:
+The blockquote is limited to [quoting text someone or somewhere](/wiki/Article_styling_criteria/Writing#block-quotation). It must not be used to format text otherwise.
```

Comment:
```suggestion
The blockquote is limited to [quoting text from someone or somewhere](/wiki/Article_styling_criteria/Writing#block-quotation). It must not be used to format text otherwise.
```

Reply:
or
```suggestion
The blockquote is limited to [quoting someone or something](/wiki/Article_styling_criteria/Writing#block-quotation). It must not be used to format text otherwise.
```